tests/scenario/test_a_charm_tracer_multi_import_warning.py

```diff
@@ -0,0 +1,67 @@
+# WARNING keep this test module on top of the scenario tests, else the
+# imports from previous tests will pollute the sys.modules and cause this test to fail.
```

Comment:
by "on top of" do you mean separate?  Like if we merged these tests into the existing other scenario test files, we'd have a conflict?  If yes, what about rewording:

```suggestion
# WARNING keep these tests separate to other scenario tests , else the
# imports from previous tests will pollute the sys.modules and cause this test to fail.
```

Reply:
no I mean that the file name starts with an "a" so that it's lexicographically first and is executed before any other scenario test. Will rephrase.